include/sentry.h

```diff
@@ -1226,6 +1226,51 @@ SENTRY_EXPERIMENTAL_API void sentry_options_set_traces_sample_rate(
 SENTRY_EXPERIMENTAL_API double sentry_options_get_traces_sample_rate(
     sentry_options_t *opts);
 
+/* -- Performance Monitoring/Tracing APIs -- */
+
+/**
+ * Constructs a new transaction context to be passed
+ * into `sentry_start_transaction`.
```

Comment:
could we describe what the name parameter is?  something like
```
*name* is the transaction name, this describes the transaction and can also be used
to search for the transaction in sentry.
```

Reply:
i'm a little reluctant to provide too much product guidance here as existing documentation tends to defer to documentation on docs.sentry.io on that front (ie `sentry_value_new_event`, `sentry_value_new_message_event`, `sentry_value_new_exception`, etc), preferring to mostly explain implementation-specific details. i'll admit that we can't really do the same for performance monitoring as its documentation isn't compiled in one easily referenceable location in the same way other types of messages are, making it difficult for us to do the same.

i've pushed an attempt to reach some sort of compromise between a full explanation and the approach used by existing documentation in #619.